README.md

```
@@ -1,36 +1,454 @@
+
+## 📋 목차
+
+1. [이 앱에 대하여](#이앱에-대하여)
+2. [API 관리방안](#api-관리방안)
+3. [i18n 적용방안](#i18n-적용방안)
+
+---
+
+# 이 앱에 대하여
+
+### 프로젝트 여는 법
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)